### PR TITLE
feat(config): add Dawon PM-S140-ZW, PM-S340-ZW and KR frequencies

### PR DESCRIPTION
**Pull request by techinpark**
The 2-button device is registered,  but the 3-button device is missing, so we created a PR to add it. 

#5136 Related PR is here

### CODE REVIEW

File: packages/config/config/devices/0x018c/pm-240-zw-3button.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  "manufacturer": "Dawon",
+  "manufacturerId": "0x018c",
+  "label": "PM-S240-ZW",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x018c/pm-240-zw-3button.json]: I think this is supposed to be
```suggestion
  "label": "PM-S340-ZW",
```
at least according to [this comment](https://github.com/zwave-js/node-zwave-js/issues/5132#issuecomment-1264405046) I apparently never followed up.

Can this be?

Comment (AlCalzone) [reply packages/config/config/devices/0x018c/pm-240-zw-3button.json]: If that's the case, I suppose we should also add the other product types mentioned in https://github.com/zwave-js/node-zwave-js/issues/5132#issuecomment-1264369553

**Review by AlCalzone — CHANGES_REQUESTED** [review]

File: packages/config/config/devices/0x018c/pm-240-zw-3button.json (hunk 1 above)
Comment (techinpark) [reply packages/config/config/devices/0x018c/pm-240-zw-3button.json]: @AlCalzone  you right! 3 button type model is `PM-S340-ZW` 👍 [6b2846c](https://github.com/zwave-js/node-zwave-js/pull/5833/commits/6b2846c888fc1670d525f4b9ebc76d18fe59cf3b) fixed here

**Comment by AlCalzone** [comment]
Thanks! I'll hijack this PR to also add the 1-button variant and the other device IDs

**Review by AlCalzone — DISMISSED** [review]

**Review by AlCalzone — APPROVED** [review]

**Pull request merged**